vars/pipelineVegavisorRestartNetwork.groovy

```diff
@@ -337,6 +357,24 @@ void call() {
                     }
                 }
             }
+            stage('Update faucet & wallet') {
+                when {
+                    expression { params.VEGA_VERSION }
+                }
+                steps {
+                    script {
+                        ['vegawallet', 'faucet'].each { app ->
```

Comment:
We need to wait for them as well because if we do not wait for them market creation is failing because wallet can be killed dead any time and some requests are failing

Reply:
I've started using different method, should be godo now